### DISCUSSION TITLE
check text

### DISCUSSION THREAD

**Discussion by ptrsk** [freeplane/freeplane#447] [General]
1. Some words cannot be translated.
Resources_xx.properties Please include it in the translation file.
Scale UI Fonts
I only found this one, but I haven't found the rest yet.

Version: 1.10.2

2. Please remove duplicate text.
Some texts were found to be duplicated.

See No. 3 below

3. Make sure that the following text is represented correctly.

msgctxt "OptionPanel.separator.outline_view"
msgid "Outline view" -> View outline

msgctxt "simplyhtml.layoutTabTitle"
msgid "Layout view" -> View layout

msgctxt "ViewLayoutTypeAction.OUTLINE.text"
msgid "Outline view" -> View outline

msgctxt "xslt_export_not_possible"
msgid "Freeplane XSLT export not possible"  -> Unable to export Freeplane XSLT

**Comment by ldmpub** [comment]
Hi and welcome 👋.

Freeplane translations are managed here: https://hosted.weblate.org/projects/freeplane/translation/

Welcome again,

**Comment by ldmpub** [comment]
and if you some strings or words can not be translated "at all" and needed to be ... I let some dev give you the appropriate answer here 😉.

**Comment by dpolivaev** [comment]
Option "Scale UI Fonts" and corresponding translations have been removed.
Always try the latest preview version before committing any bugs.

Some strings are duplicated because of technical reasons. I am not going to fix it.

I do not understand what you mean by saying "Make sure that the following text is represented correctly". I believe that all texts are displayed properly, aren't they?

In general, I would appreciate it if a separate issue is created for each separate topic. Actually, issues related to more than one topic should be closed without any further action because it is hard to track them otherwise.

**Comment by ptrsk** [comment]
3  Additional explanation
If you want to find the location in the translation file, please search for it.

The question in 3 was to ask if the original text needs to be modified.
Because the translation of this part in some languages doesn't fit cleanly.
I'm asking if the word order is correct.
The reason why I'm not sure is because I don't know when these functions are used.

**Reply by euu2021** [reply]
Just to confirm if I understood your question 3 correctly: 

When you write:

`msgid "Outline view" -> View outline`

It means that `Outline view` is how the text is currently written in Freeplane, and `View outline` is how you suggest that it should be written?

**Comment by dpolivaev** [comment]
I converted the issue into discussion so that other community members can contribute.
If the community suggests any changes I can implement them.
